src/classes/frDonation.cls

```diff
@@ -338,7 +407,7 @@ public class frDonation extends frModel implements frSyncable {
 
     ///CONSTANTS///
 
-    public static final String TYPE = 'Donation';
+    public static final String TYPE = frUtil.hasNPCobjects() ?  'Gift Transaction' : 'Donation';
```

Comment:
Is there a need for this value to be different based on if the NPC objects are present or not?  In my understanding there isn't a way to convert an existing Salesforce org between NPC vs non-NPC, so the mappings belong to donations, not Opportunity/GiftTransaction.  I think removing this can simplify a lot of the mapping and frSetupController code

Reply:
This could be fixed but some of the tests/ functionality is tied to this so may take some time to revert it.

Reply:
I think both of the TYPE variables should not be changed for this pull request, since these are not used in queries and previously were not `Opportunity` nor `Contact` I don't think they need to correspond with the SObject